aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -459,3 +471,48 @@ proptest! {
         );
     }
 }
+
+#[test]
+#[should_panic]
+fn test_aggregator_snapshot_not_equivalent_gas() {
+    let test_env = TestEnvConfig {
+        executor_mode: ExecutorMode::BothComparison,
+        aggregator_execution_mode: AggregatorMode::BothComparison,
+        block_split: BlockSplit::Whole,
+    };
+
+    println!("Testing test_aggregator_snapshot {:?}", test_env);
```

Comment:
This prints based on --no-capture, right?

Reply:
yup